packages/vue/src/components/search/SearchBox.jsx

```diff
@@ -198,12 +199,29 @@ const SearchBox = defineComponent({
 				});
 			}
 
-			suggestionsArray = suggestionsArray.map((s) => {
-				if (s.sectionId) {
-					return s;
-				}
-				return { ...s, sectionId: s._suggestion_type };
-			});
+			suggestionsArray
+			= suggestionsArray
+					.map((s) => {
+						if (s.sectionId) {
+							return s;
+						}
+						return { ...s, sectionId: s._suggestion_type };
+					})
+					.map((suggestion) => {
+						if (suggestion._suggestion_type === 'document') {
```

Comment:
@SavvyShah Let's have an enum map for storing suggestion types for SearchBox and it should sit in reactivecore.

Reply:
OK.